lib/collections/Simulation.js

```diff
@@ -34,6 +34,22 @@ SimulationSchema = new SimpleSchema({
 
 Simulation.attachSchema(SimulationSchema);
 
+Simulation.after.insert((userId, simulation) => {
```

Comment:
This file will be loaded client and server side as it is in `./lib`, thus the hook will be executed on both side as stated in the collection-hooks package. You should only include the hook on one side (preferably server)